libs/core/src/components/pds-input/readme.md

```diff
@@ -49,6 +49,16 @@ Type: `Promise<void>`
 
 
 
+## Shadow Parts
+
+| Part        | Description |
+| ----------- | ----------- |
+| `"append"`  |             |
+| `"prefix"`  |             |
+| `"prepend"` |             |
+| `"suffix"`  |             |
```

Comment:
These need descriptions, which can be added at the top of the `input.tsx` file using TSDoc syntax.